dotCMS/src/main/java/com/dotcms/rest/api/v1/secret/view/ServiceIntegrationView.java

```diff
@@ -0,0 +1,64 @@
+package com.dotcms.rest.api.v1.secret.view;
+
+import com.dotcms.security.secret.ServiceDescriptor;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import java.util.List;
+
+public class ServiceIntegrationView {
+
+    private final long configurationsCount;
+
+    private final String key;
+
+    private final String name;
+
+    private final String description;
+
+    private final String iconUrl;
+
+    @JsonInclude(Include.NON_NULL)
+    private final List<SiteView> sites;
+
+    public ServiceIntegrationView(final ServiceDescriptor serviceDescriptor, final long configurationsCount) {
+        this.key = serviceDescriptor.getKey();
+        this.name = serviceDescriptor.getName();
+        this.description = serviceDescriptor.getDescription();
+        this.iconUrl = serviceDescriptor.getIconUrl();
+        this.configurationsCount = configurationsCount;
+        this.sites = null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assigning an Object to null is a code smell.  Consider refactoring.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4623512)